meetings/conferences/serializers.py

```diff
@@ -49,4 +49,4 @@ def get_my_submission_count(self, obj):
     def get_can_edit(self, obj):
         request = self.context.get('request')
         user = User.objects.get(username=request.user)
-        return user == obj.admin
+        return user.username == obj.admin.username
```

Comment:
Yeah, it needed to be this way to pass the test. Usernames are unique fields so it shouldn't make a difference.